test/k8sT/hubble.go

```diff
@@ -180,28 +145,36 @@ var _ = Describe("K8sHubbleTest", func() {
 		})
 
 		It("Test L3/L4 Flow", func() {
+			ctx, cancel := context.WithTimeout(context.Background(), helpers.MidCommandTimeout)
+			defer cancel()
+			follow := kubectl.HubbleObserveFollow(ctx, hubbleNamespace, hubblePodK8s1, fmt.Sprintf(
+				"--last 1 --type trace --from-pod %s/%s --to-namespace %s --to-label %s --to-port %d",
```

Comment:
do you want `--last 1` with follow?

Reply:
This is intentional, yes. It's supposed to address a potential race:

The `hubble observe --follow` process is started in the background. Therefore, there is a small possibility that we proceed with the curl request before the `--follow` process in the background is actually receiving flows yset. Therefore, `--follow` would miss the curl request, causing the test to fail. By introducing `--last`, we don't care which happens first: If the curl request happened before, we can still get it from the ring buffer thanks to `--last`, if the curl request happend later, then we will receive it thanks to follow-mode.